lib/t/push_notifications/apns.ex

```diff
@@ -280,6 +280,41 @@ defmodule T.PushNotifications.APNS do
     base_alert_payload(type, alert, %{})
   end
 
+  # newbies alerts
+
+  # maybe have some play on `orientation` (as in uni)
+  # 3. familiarization with something: many judges give instructions to assist jury orientation.
+  #    • (also orientation course) _mainly North American_ a course giving information to
+  #                                newcomers to a university or other institution.
+
+  def build_alert_payload("newbie_live_mode_today" = type, _data) do
+    alert = %{
+      # like Since Live orientation today
+      "title" => dgettext("apns", "Since LIVE today 🥳"),
+      "body" => dgettext("apns", "Come to the party at 19:00, it will be 🔥")
+    }
+
+    base_alert_payload(type, alert, %{})
+  end
+
+  def build_alert_payload("newbie_live_mode_soon" = type, _data) do
+    alert = %{
+      "title" => dgettext("apns", "Since LIVE starts soon 🔥"),
+      "body" => dgettext("apns", "Come chat with new people 🥳")
+    }
+
+    base_alert_payload(type, alert, %{})
+  end
+
+  def build_alert_payload("newbie_live_mode_started" = type, _data) do
+    alert = %{
+      "title" => dgettext("apns", "Since Live starts 🥳"),
+      "body" => dgettext("apns", "Only you and other new users are invited 🎉")
+    }
+
+    base_alert_payload(type, alert, %{})
+  end
```

Comment:
I guess the push notification massages need to be updates slightly to mention that it's for newbies. I've updated only the `newbie_live_mode_started` one, but it's not good. I'll think a bit and post some variants here.